fragments/filter/README.md

```diff
@@ -0,0 +1,786 @@
+      "type": "application/geo+json"
+    },
+    {
+      "title": "Queryables",
+      "href": "https://example.org/queryables",
+      "rel": "http://www.opengis.net/def/rel/ogc/1.0/queryables",
+      "type": "application/schema+json"
+    }
+  ],
+  "stac_extensions": [],
+  "stac_version": "1.0.0",
+}
+```
+
+A client can use the link with `"rel": "http://www.opengis.net/def/rel/ogc/1.0/queryables"` to retrieve the queryables.
+
+The Queryables endpoint (`/queryables`) returns something like the following:
+
+```json
+{
+  "$schema" : "https://json-schema.org/draft/2019-09/schema",
+  "$id" : "https://example.org/queryables",
+  "type" : "object",
+  "title" : "Queryables for Example STAC API",
+  "description" : "Queryable names for the example STAC API Item Search filter.",
+  "properties" : {
+    "id" : {
+      "description" : "ID",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/id"
+    },
+    "collection" : {
+      "description" : "Collection",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/collection"
+    },
+    "geometry" : {
+      "description" : "Geometry",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/geometry"
+    },
+    "datetime" : {
+      "description" : "Datetime",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/datetime.json#/properties/datetime"
+    },
+    "eo:cloud_cover" : {
+      "description" : "Cloud Cover",
+      "$ref": "https://stac-extensions.github.io/eo/v1.0.0/schema.json#/properties/eo:cloud_cover"
+    },
+    "gsd" : {
+      "description" : "Ground Sample Distance",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/instrument.json#/properties/gsd"
+    },
+    "assets_bands" : {
+      "description" : "Asset eo:bands common names",
+      "$ref": "https://stac-extensions.github.io/eo/v1.0.0/schema.json#/properties/eo:bands/common_name"    
+    }
+  }
+}
+```
+
+Alternately, the client could retrieve the queryables for a single collection with 
+`/collections/collections1/queryables`, which may have more queryables than available for the entire catalog, since
+there may be queryables that are only relevant to that collection. 
+
+Notice in this schema that instead of directly defining the type information about each field, we have instead used the JSON 
+Schema `$ref` mechanism to refer to a STAC schema definition. This not only allows the reuse of these JSON Schema definitions,
+but also binds an arbitrarily-named Queryable to a specific STAC field. For example, in the above we know that the 
+`eo:cloud_cover` field is referring to the field of the same name in the EO Extension not because they happen to have the same 
+name, but rather because the `$ref` indicates it. The field could just as well be named "cloud_cover", "CloudCover", or "cc", 
+and we would still know it filtered on the EO extension `eo:cloud_cover` field.
```

Comment:
I believe this is the case, but wasn't clear to me on the first readthrough - in a CQL query (either text or JSON), the arbitrarily named Queryable is used in the query rather then a reference to the fully qualified property. E.g. in the case it was called "CloudCover", you'd say "CloudCover <- 20.0". Is this correct?

Reply:
Yes, and I've clarified it.